metricbeat/module/elasticsearch/cluster_stats/data.go

```diff
@@ -215,7 +216,13 @@ func getClusterMetadataSettings(httpClient *helper.HTTP) (mapstr.M, error) {
 	return clusterSettings, nil
 }
 
-func eventMapping(r mb.ReporterV2, httpClient *helper.HTTP, info elasticsearch.Info, content []byte, isXpack bool) error {
+func eventMapping(
+	r mb.ReporterV2,
+	httpClient *helper.HTTP,
+	info elasticsearch.Info,
+	content []byte,
+	isXpack bool,
+	log *logp.Logger) error {
```

Comment:
I noticed elsewhere that unused function parameters were named `_`. 🤷‍♂️

Reply:
I'm surprised the linter isn't complaining about the unused parameter. Can we change this to

```suggestion
	_ *logp.Logger) error {
```